src/call_builder.ts

```diff
@@ -202,6 +206,17 @@ export class CallBuilder<
     return this;
   }
 
+  /**
+   * Sets `join` parameter for the current call. Returns the CallBuilder object
+   * on which this method has been called.
+   * @param {"transactions"} join Records to be included in the response.
+   * @returns {object} current CallBuilder instance.
+   */
+  public join(include: "transactions"): this {
```

Comment:
This is a string literal, it will force this value to be `transactions` -- Once join supports more types and we know the query param shape, we'll be able to add more validations into this.

Reply:
@morleyzhi yeah it does, if there is an error then it will bubble up from the server and handled by our error handling. Also if you are in TS --then you will be allowed to pass only "transactions" which is the only valid value for now.